pages/components/attention.html

```diff
@@ -170,25 +170,25 @@ <h2 class="mt-24 mb-16">Accessibility</h2>
         </p>
         <p>
           Attention element handles accessibility automatically by wrapping its slotted content with
-          a <code>div</code> with <code>role="tooltip"</code>, and setting an
-          <code>aria-describedby</code> attribute on the target element.
+          a <code>div</code> with either <code>role="tooltip"</code> for tooltip or <code>role="img"</code> for callout and popover, a default <code>aria-label</code>and setting an
+          <code>aria-details</code> attribute on the target element. The default <code>aria-label</code> also supports i18n.
         </p>
         <p>
           It is possible to tell assistive technologies to recognize only a part of Attention's text
-          content. To do that set <code>role="tooltip"</code> on the relevant text element nested in
+          content. To do that set the <code>role</code> and <code>aria-label</code> attributes on the relevant text element nested in
           <code>w-attention</code> and reference it by id through the use of
-          <code>aria-describedby</code>. The <code>aria-describedby</code> attribute is on the
+          <code>aria-details</code>. The <code>aria-details</code> attribute is on the
           target element, not on <code>w-attention</code>.
         </p>
 
         <syntax-highlight>
           <w-attention placement="top" popover>
             <div slot="message">
-              <p id="aria-content" role="tooltip">I'm a popover with ARIA "tooltip" role</p>
+              <p id="aria-content" role="tooltip" aria-label="popover with ARIA 'tooltip' role">I'm a popover with ARIA "tooltip" role and a aria-label</p>
```

Comment:
Same here, role="tooltip" will tell screen readers that it should read whatever text this paragraph holds. Does that make sense?